astroquery/esa/jwst/core.py

```diff
@@ -965,10 +965,11 @@ def get_obs_products(self, *, observation_id=None, cal_level="ALL",
             composite products based on level 2 products). To request upper
             levels, please use get_related_observations functions first.
             Possible values: 'ALL', 3, 2, 1, -1
-        product_type : str, optional, default None
-            List only products of the given type. If None, all products are \
-            listed. Possible values: 'thumbnail', 'preview', 'auxiliary', \
-            'science'.
+        product_type : str or list, optional, default None
+            List only products of the given type. If None, empty or an element
+            of the list is empty, all products are listed.
+            Possible values: 'thumbnail', 'preview', 'auxiliary',
+            'science', 'info', ['science', 'preview']...
```

Comment:
are there any more options? If yes, and we know them, then please list, otherwise drop the `...`. No need to allow an empty list, the default `None` should cover that use case.

Reply:
I used ... because in the list you can provide any combination of 1, 2, 3, 4 or 5 elements, ['science', 'preview'] is just one example of all of them. If you want we can better express this idea.